src/components/step/ConnectDeviceAccountStepContainer/ConnectDeviceAccountStepContainer.tsx

```diff
@@ -2,17 +2,18 @@ import React, { useState, useEffect } from "react";
 import MyDataHelps, { StepConfiguration } from "@careevolution/mydatahelps-js";
 import ConnectDeviceAccountStep from "../ConnectDeviceAccountStep";
 import { getFitbitProviderID, getGarminProviderID, getOmronProviderID } from "../../../helpers/providerIDs";
+import { deviceType } from "../../../helpers/deviceType";
 
 export interface ConnectDeviceAccountStepContainerProps {
-    deviceType?: string;
+    deviceType: deviceType
```

Comment:
@bxh why is this required now?  Isn't device type set by step configuration?  So what should this be set to?

Reply:
To be clear, view library isn't building because we put this step container in view library.  It should be inferring it's configuration from the SDK call to get configuration, we don't set device type in the view library

Reply:
`deviceType` is `Fitbit`, `Garmin`, or `Omron`. Do you mean the value can be read from the step configuration?

Reply:
The value is always supposed to be read from step configuration...so it doesn't make sense to make this prop required

Reply:
OK, can I fix it next week?

Reply:
I already fixed it: https://github.com/CareEvolution/MyDataHelpsUI/pull/270